packages/cli/index.js

```diff
@@ -183,21 +183,24 @@ async function prepareMinimalAppEnv (appName, dbName, redisName, port, sport, se
     await updateAppEnvFile(appName, 'COOKIE_SECRET', `${appName}-secret`)
 }
 
-async function prepareAppEnvLocalAdminUsers (appName) {
+async function prepareAppEnvLocalAdminUsers (appName, identity = 'email') {
     // TODO(pahaz): check create-user.js logic!
-    const justUserEmail = await getAppEnvValue(appName, 'DEFAULT_TEST_USER_IDENTITY') || 'user@example.com'
+    const justUserDefaultIdentity = identity === 'email' ? 'user@example.com' : '+79067777777'
```

Comment:
It's better to avoid any hardcoded credentials

Reply:
It's the same behaviour as was before.
Before: 
1. took email from `.env` or use default
Now:
1. If auth is email use this default, or this default otherwise
2. Use identity from `.env` or use default